pkg/model/components/kubelet.go

```diff
@@ -68,6 +69,24 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
+	if kubernetesVersion.Major == 1 && kubernetesVersion.Minor >= 4 {
+		// For pod eviction in low memory or empty disk situations
+		if options.Kubelet.EvictionHard == nil {
+			evictionHard := []string{
+				// TODO: Some people recommend 250Mi, but this would hurt small machines
+				"memory.available<100Mi",
+
+				// Disk eviction (evict old images)
+				// We don't need to specify both, but it seems harmless / safer
+				"nodefs.available<10%",
```

Comment:
I'm trying to mirror the GCE configuration:
https://github.com/kubernetes/kubernetes/blob/f18a921a0308c4ecd2e7e8b7e30e39a8216a3447/cluster/gce/config-default.sh#L184

I just wasn't sure about the free memory - I'm pretty sure that would break anyone running on smaller instances, so I stuck with the 100Mi default.